Workbooks/AKS/Syslog/Syslog.workbook

```diff
@@ -1,580 +1,567 @@
+									"representation": "4",
+									"text": "{0}{1}"
+								},
+								{
+									"operator": "==",
+									"thresholdValue": "crit",
+									"representation": "critical",
+									"text": "{0}{1}"
+								},
+								{
+									"operator": "Default",
+									"thresholdValue": null,
+									"representation": "Normal",
+									"text": "{0}{1}"
+								}
+							]
+						}
+					},
+					"leftContent": {
+						"columnMatch": "count_SeverityLevel",
+						"formatter": 12,
+						"formatOptions": {
+							"palette": "auto"
+						},
+						"numberFormat": {
+							"unit": 17,
+							"options": {
+								"maximumSignificantDigits": 3,
+								"maximumFractionDigits": 2
+							}
+						}
+					},
+					"secondaryContent": {
+						"columnMatch": "Trend",
+						"formatter": 21,
+						"formatOptions": {
+							"min": 0,
+							"max": 5000,
+							"palette": "green"
+						}
+					},
+					"showBorder": true,
+					"sortCriteriaField": "count_SeverityLevel",
+					"sortOrderField": 2
+				}
+			},
+			"name": "Syslog Facility tiles"
+		},
+		{
+			"type": 3,
+			"content": {
+				"version": "KqlItem/1.0",
+				"query": "Syslog\r\n| where HostName in~ ({HostName}) or '*' in~ ({HostName})\r\n| where Facility in~ ({Facility}) or '*' in~ ({Facility})\r\n| where SeverityLevel in~ ({SeverityLevel}) or '*' in~ ({SeverityLevel})\r\n| summarize SyslogCount=count(SeverityLevel) by Facility, HostName\r\n| extend jkey = 1\r\n| join (Syslog\r\n        | where HostName in~ ({HostName}) or '*' in~ ({HostName})\r\n        | where Facility in~ ({Facility}) or '*' in~ ({Facility})\r\n        | where SeverityLevel in~ ({SeverityLevel}) or '*' in~ ({SeverityLevel})\r\n        | make-series SyslogTimeLine = count(SeverityLevel) default = 0 on TimeGenerated from {timeRange:start} to {timeRange:end} step {timeRange:grain} by Facility) on Facility\r\n| project-away jkey, Facility1, TimeGenerated",
+				"size": 0,
+				"title": "Message counts by node and facility",
+				"timeContextFromParameter": "timeRange",
+				"queryType": 0,
+				"resourceType": "microsoft.operationalinsights/workspaces",
+				"crossComponentResources": [
+					"{Workspace}"
+				],
+				"gridSettings": {
+					"formatters": [{
+							"columnMatch": "Facility",
+							"formatter": 5,
+							"formatOptions": {}
+						},
+						{
+							"columnMatch": "SyslogCount",
+							"formatter": 8,
+							"formatOptions": {
+								"palette": "blue"
+							}
+						},
+						{
+							"columnMatch": "SyslogTimeLine",
+							"formatter": 21,
+							"formatOptions": {
+								"min": 0,
+								"max": 1000,
+								"palette": "green"
+							}
+						},
+						{
+							"columnMatch": "EventTimeLine",
+							"formatter": 21,
+							"formatOptions": {
+								"min": 0,
+								"max": 500,
+								"palette": "green"
+							}
+						}
+					],
+					"rowLimit": 50,
+					"hierarchySettings": {
+						"treeType": 1,
+						"groupBy": [

```

Comment:
Can we make the title to show 'Node' instead of HostName to keep it consistent with the parameter?